### PR TITLE
List of components & their current status

### PR DESCRIPTION
**Pull request by hopkins-tk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
 # Blaze Elements
 
 Web Component Blaze Elements
+
+http://blazecss.com/components/buttons/
```
Comment (devopsmariocom) [inline README.md:5]: please fix link

File: README.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+| Ranges | DONE |
+| Tables | - |
+| Tabs | - |
+| Tags | - |
+| Toggles | DONE |
+| Toasts | - |
+| Tooltips | DONE |
+| Trees | DONE |
+| Typography | - |
+| - Link | DONE |
```
Comment (devopsmariocom) [inline README.md:49]: why is there dash?

File: packages/alert/Alert.tsx [hunk 3]
```diff
@@ -47,7 +47,7 @@ export class Alert extends Component<AlertProps> {
     return [
       <style>{styles}</style>,
       isOpen && <div className={className}>
-        <Button type="close" onClick={this.close}>×</Button>
+        <Button close onClick={this.close}>×</Button>
```
Comment (devopsmariocom) [inline packages/alert/Alert.tsx:50]: let't try from now on strictly separate PR 😉

File: README.md (hunk 2 above)
Comment (hopkins-tk) [reply README.md:49]: It is here as it is part of Typography package: http://blazecss.com/components/typography/

File: README.md (hunk 1 above)
Comment (hopkins-tk) [reply README.md:5]: There is no such link :(

File: packages/alert/Alert.tsx (hunk 3 above)
Comment (hopkins-tk) [reply packages/alert/Alert.tsx:50]: You are right - slipped...

**Pull request merged**